browser/app/js/objects/actions.js

```diff
@@ -325,7 +325,6 @@ const downloadZip = (url, req, dispatch) => {
 
   xhr.onload = function(e) {
     if (this.status == 200) {
-      dispatch(resetCheckedList())
```

Comment:
Do we want to not clear checked objects on download? If this is the case we can keep this change as is.